node/resolvers/Queries/Organizations.ts

```diff
@@ -9,6 +10,16 @@ import {
 import type { Organization } from '../../typings'
 import GraphQLError, { getErrorMessage } from '../../utils/GraphQLError'
 import checkConfig from '../config'
+import { organizationStatus } from '../fieldResolvers'
+
+export interface GetOrganizationsByEmailWithStatus {
```

Comment:
sorry, I'm a bit confused. If this an interface, I think it can live in the `typings.ts` file (without the `export`) and include it here on the `import type { Organization } from '../../typings'` line (same as `Organization` that is also an interface)